examples/generated/test/react-native/logic/Optionals.js

```diff
@@ -0,0 +1,70 @@
+import React from "react"
+import { Text, View, StyleSheet } from "react-native"
+
+import colors from "../colors"
+import shadows from "../shadows"
+import textStyles from "../textStyles"
+
+export default class Optionals extends React.Component {
+  render() {
+
+    let Label$text
+    let StringParam$text
+    let View$backgroundColor
+    let unwrapped
+    Label$text = ""
+    StringParam$text = "No string param"
+    View$backgroundColor = "transparent"
+
+    if (this.props.boolParam == true) {
+      Label$text = "boolParam is true"
+      View$backgroundColor = colors.green200
+    }
+    if (this.props.boolParam == false) {
+      Label$text = "boolParam is false"
+      View$backgroundColor = colors.red200
+    }
+    if (this.props.boolParam == null) {
```

Comment:
Using `==` instead of `===` here makes it so that `undefined` will work too, for when a component is consumed outside of Lona and _no props_ is passed.